### PR TITLE
Feature namespace refactor shader refactor

### PR DESCRIPTION
**Pull request by DDReaper**
Overview
---
Updated shaders to remove all instances of HoloToolkit (and TLA's)

(Someone already changed the shader names??)

Also fixed the ProjectSettings asset to change the Toolkit name there as well.

Changes
---

[x] Updated Shaders
[x] Removed obsolete shaders except unlit

### CODE REVIEW

File: ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -153,7 +153,7 @@ PlayerSettings:
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
     Android: com.Company.ProductName
-    Standalone: Microsoft.HoloToolkit-Unity
+    Standalone: .HoloToolkit-MixedRealityToolkit
```
Comment (keveleigh) [inline ProjectSettings/ProjectSettings.asset]: Were these changed intentionally?

File: Assets/MixedRealityToolkit/Common/Shaders/MixedRealityToolkitCommon.cginc [hunk 2]
```diff
@@ -1,8 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-#ifndef HOLOTOOLKIT_COMMON
-#define HOLOTOOLKIT_COMMON
+#ifndef MIXEDREALITYTOOLKIT_COMMON
+#define MIXEDREALITYTOOLKIT_COMMON
+
+#include "UnityCG.cginc"
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Common/Shaders/MixedRealityToolkitCommon.cginc]: Do we need this new `#include`, though we didn't have it before?

File: ProjectSettings/ProjectSettings.asset (hunk 1 above)
Comment (StephenHodgson) [reply ProjectSettings/ProjectSettings.asset]: Yeah it should probably just have the Holo replaced with MixedReality

File: Assets/MixedRealityToolkit/Common/Shaders/MixedRealityToolkitCommon.cginc (hunk 2 above)
Comment (DDReaper) [reply Assets/MixedRealityToolkit/Common/Shaders/MixedRealityToolkitCommon.cginc]: Strangely enough, Yes. Wouldn't compile without it.  Very odd.

File: ProjectSettings/ProjectSettings.asset (hunk 1 above)
Comment (DDReaper) [reply ProjectSettings/ProjectSettings.asset]: I changed it in the Editor.  Looks like Unity is still retaining the original project name in the asset.  I'll fix that.

File: Assets/MixedRealityToolkit/Common/Shaders/MixedRealityToolkitCommon.cginc (hunk 2 above)
Comment (keveleigh) [reply Assets/MixedRealityToolkit/Common/Shaders/MixedRealityToolkitCommon.cginc]: Huh, strange!

File: ProjectSettings/ProjectSettings.asset [hunk 3]
```diff
@@ -12,7 +12,7 @@ PlayerSettings:
   useOnDemandResources: 0
   accelerometerFrequency: 60
   companyName: Microsoft
-  productName: MixedRealityToolkit-Unity
+  productName: MixedRealityToolkit
```
Comment (keveleigh) [inline ProjectSettings/ProjectSettings.asset]: We should probably leave this with the `-Unity`, to match the repo name.

File: ProjectSettings/ProjectSettings.asset [hunk 4]
```diff
@@ -153,7 +153,7 @@ PlayerSettings:
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
     Android: com.Company.ProductName
-    Standalone: Microsoft.HoloToolkit-Unity
+    Standalone: .MixedRealityToolkit
```
Comment (keveleigh) [inline ProjectSettings/ProjectSettings.asset]: We should probably leave this with the `-Unity`, to match the repo name. Was dropping the `Microsoft` intentional as well?

File: Assets/MixedRealityToolkit/Common/Shaders/Obsolete/HoloToolkitCommon.cginc [hunk 5]
```diff
@@ -1,35 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-#include "UnityCG.cginc"
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/Common/Shaders/Obsolete/HoloToolkitCommon.cginc]: Ahh, looks like there were two HoloToolkitCommon.cginc files, one in the Obsolete folder and one in the FastConfigurable folder, and this one has that `#include`.

File: ProjectSettings/ProjectSettings.asset (hunk 4 above)
Comment (DDReaper) [reply ProjectSettings/ProjectSettings.asset]: Good point, I'll put the Microsoft bit back in and the suffix and then merge.

File: Assets/MixedRealityToolkit/Common/Shaders/Obsolete/HoloToolkitCommon.cginc (hunk 5 above)
Comment (DDReaper) [reply Assets/MixedRealityToolkit/Common/Shaders/Obsolete/HoloToolkitCommon.cginc]: That explains why it wasn't picked up before #Phew

**Pull request merged**